From b1d531bfb130e7149ffe24bb0f1c5c01f20c3a5f Mon Sep 17 00:00:00 2001
From: Mario Emmenlauer <mario@emmenlauer.de>
Date: Mon, 29 Jun 2020 16:15:14 +0200
Subject: [PATCH 23/24] CMakeLists.txt: Export all symbols (#136)

---
 CMakeLists.txt | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a51add2..46e3abf 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,10 @@
 cmake_minimum_required(VERSION 3.0)
 project(double-conversion VERSION 3.1.5)
 
+if(BUILD_SHARED_LIBS AND MSVC)
+  set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+endif()
+
 set(headers
     double-conversion/bignum.h
     double-conversion/cached-powers.h
-- 
2.29.2

